performance-tests/src/libs/backbone-client/identity.ts

```diff
@@ -0,0 +1,65 @@
+import { b64encode } from "k6/encoding";
+import { Response } from "k6/http";
+import { ChallengeResponse } from "../../models/challenge";
+import { CreateIdentityRequest, CreateIdentityResponse } from "../../models/identity";
+import { TokenResponse } from "../../models/token";
+import { ChallengeRequestRepresentation, CryptoHelper } from "../crypto-helper";
+import { Httpx } from "https://jslib.k6.io/httpx/0.1.0/index.js";
+
+export function createIdentity(client: Httpx, clientId: string, clientSecret: string): { httpResponse: Response; generatedPassword: string } {
+    try {
+        const keyPair = CryptoHelper.generateKeyPair();
+
+        const challenge = getChallenge(client);
+
+        const signedChallenge = CryptoHelper.signChallenge(keyPair, challenge);
+
+        const generatedPassword = CryptoHelper.generatePassword()!;
+
+        const createIdentityRequest: CreateIdentityRequest = {
+            clientId: clientId,
+            clientSecret: clientSecret,
+            signedChallenge: { challenge: JSON.stringify(challenge), signature: b64encode(JSON.stringify(signedChallenge)) },
+            identityPublicKey: b64encode(JSON.stringify(keyPair.pub)),
+            devicePassword: generatedPassword,
+            identityVersion: 1
+        };
+
+        const httpResponse = client.post("Identities", JSON.stringify(createIdentityRequest), {
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            headers: { "Content-Type": "application/json" }
+        }) as Response;
+        return { httpResponse, generatedPassword };
+    } catch (e) {
+        console.error(e);
+        throw e;
+    }
+}
+
+export function exchangeToken(client: Httpx, createdIdentityResponse: CreateIdentityResponse, password: string) : TokenResponse {
```

Comment:
Why are you passing the whole `CreateIdentityResponse` when you only need the username?